scenes/game_elements/characters/npcs/talker/components/talker.gd

```diff
@@ -22,6 +22,8 @@ func _ready() -> void:
 		return
 	interact_area.interaction_started.connect(_on_interaction_started)
 	DialogueManager.dialogue_ended.connect(_on_dialogue_ended)
+	if npc_name:
+		interact_area.action = "Talk to %s" % npc_name
```

Comment:
I can't remember when it was decided to have the NPC names in it, and the issue doesn't tell. Personally I think that it would be nice after the player meets the NPC. The elders are not presenting themselves as "hi, I'm lore quest starter" anyways. I would ask @Stregoica777 what she thinks.